### DISCUSSION TITLE
Crossplay v2.2 for Terraria 1.4.4.9

### DISCUSSION THREAD

**Discussion by Moneylover3246** [Moneylover3246/Crossplay#57] [Announcements]
## Overview
- Introduces support for servers running on Terraria 1.4.4.9

<hr /><em>This discussion was created from the release <a href='https://github.com/Moneylover3246/Crossplay/releases/tag/'>Crossplay v2.2 for Terraria 1.4.4.9</a>.</em>

**Comment by Soundneko98** [comment]
No funciona ..... El mismo server no lo acepta......

**Comment by Astroman250** [comment]
The File Link give me a 404 error. It won't let me download it.

**Reply by yessikapenavega383-gif** [reply]
Donde puedo ver el enlace ,me lo quiero descargar pero no lo encuentro :(

**Comment by TerraKill765747646** [comment]
its not working👍

**Comment by GAMEPL4YS** [comment]
The plugin doesn't work, can you fix it?

**Reply by SirMonkeyBoy** [reply]
Please make an issue with more information

**Comment by REAPERSCRIPTS** [comment]
it says lost connection when i join on mobile

**Reply by SirMonkeyBoy** [reply]
Please make an issue with more information

**Reply by REAPERSCRIPTS** [reply]
I tried running the server on my pc it works and all but when I try to join with my phone it says found server then      
It says lost connection. I tried joining on my pc and it worked but it didn’t for mobile

**Reply by SirMonkeyBoy** [reply]
Please make an issue under issues in this repository with your logs

**Reply by REAPERSCRIPTS** [reply]
Where is the issue thing

**Reply by SirMonkeyBoy** [reply]
https://github.com/Moneylover3246/Crossplay/issues

**Comment by Whigsz** [comment]
Can 1.4.4.14 join in this?

**Reply by SirMonkeyBoy** [reply]
I'm not sure but you can try i think it should work

**Comment by Valentino621** [comment]
El link no funciona puedes poner uno nuevo? Please tanks

**Reply by SirMonkeyBoy** [reply]
Did you download the Crossplay.dll from assets

**Reply by Valentino621** [reply]
Hi, that's it, thanks, but what do I do after downloading everything? Do you have a tutorial? Because I downloaded everything but I don't know what to do?

**Reply by SirMonkeyBoy** [reply]
put it in the Tshock plugin folder then start the server

**Reply by Valentino621** [reply]
I don't know how to download Tshock

**Reply by SirMonkeyBoy** [reply]
Then you need to go get tshock from here https://github.com/Pryaxis/TShock https://ikebukuro.tshock.co/#/

**Reply by Valentino621** [reply]
do i download everything from my mobile?

**Reply by Valentino621** [reply]
does it work without a computer?

**Reply by SirMonkeyBoy** [reply]
computer recommend as tshock is made for the desktop terraria and this plugin adds support for mobile if you don't need support for desktop terraria i would you the normal mobile terraria server on the terraria website but you still need a computer for that

**Reply by Valentino621** [reply]
oh thanks sorry for bothering you

**Comment by yessikapenavega383-gif** [comment]
Hola, me quiero descargar el enlace pero no lo encuentro :(

**Comment by TerraShade** [comment]
This one worked when 1.4.4.9 was out, and i know it will not work for 1.4.5, but will there be a new version for it?
